build/commands/lib/config.js

```diff
@@ -595,6 +595,9 @@ Config.prototype.buildArgs = function () {
     args.brave_safebrowsing_api_key = this.braveAndroidSafeBrowsingApiKey
     args.safe_browsing_mode = 2
 
+    // Required since cr126 to use Chrome password store
+    args.use_login_database_as_backend = true
```

Comment:
I'm confused, why do we not want the same behavior as upstream here?

Reply:
cc: @AlexeyBarabash

Reply:
because now without that flag Chrome uses OS-provided password store. If so, we lose ability to sync passwords with our Sync, because Chrome uses Google Services and Google account to sync it.